### ISSUE TITLE
How to use `Accessibility.form`?

### ISSUE THREAD

**Issue by remibd** [tesk9/accessible-html#17]
Hi!

Thanks for your work on this package, it is very important and not highlighted enough in the Elm community :slightly_smiling_face:

Since Elm 0.19, [a few XSS protections](https://github.com/elm/virtual-dom/commit/56b5faa19fad198eb23e3c5f121c89b5dcbe7f3a) have been added to *virtual-dom*, and the following pattern for single-page applications is now disallowed :
```elm
form [ action "javascript:void(0)" ]
    [ button [ type_ "submit", onClick SubmittedForm ] [ text "Submit" ] ]
```
Clicking the button will only result in an annoying dialog box.

The accepted pattern seems to be :
```elm
form [ onSubmit SubmittedForm ]
    [ button [ type_ "submit" ] [ text "Submit" ] ]
```
but it cannot be used with `Accessibility.form` since it only takes non-interactive attributes.

Is there a way to stick with `Accessibility.form` while not using any XSS-sensitive strings? Or should we fall back to `Http.form` for single-page applications?

**Comment by tesk9** [comment]
Thanks for the issue! I wasn't aware of this. I'll look into it!
